paper/src/main/java/co/aikar/commands/PaperAsyncTabCompleteHandler.java

```diff
@@ -65,9 +63,6 @@ public void onAsyncTabComplete(AsyncTabCompleteEvent event) {
 
             BukkitCommandIssuer issuer = this.manager.getCommandIssuer(event.getSender());
             List<String> results = cmd.tabComplete(issuer, commandLabel, args, true);
-            if (event.getCompletions() instanceof ImmutableList) {
```

Comment:
Same issue occurs here, just needs to be the java version check now instead. If it returned immutable, we need to rebuild as mutable.

Reply:
FIxed